scripts/README.md

```diff
@@ -20,6 +23,23 @@ A node script to generate markdown files required for hosting Learning Path on i
 ```
 npm ci
 node generate_learning_path_markdown.js
+```
+
+## How to update innersourcecommons.org with new articles
+
+Anytime there are _new_ articles written, you need to run the script that generates the new pages on the [innersourcecommons.org] site.
+
+1. Clone the [InnerSourceCommons/InnerSourceLearningPath] and [InnerSourceCommons/innersourcecommons.org] repos.
+1. Is this the first time that articles have been written for this section in this language?
+If so, then update the ["sections" config](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/scripts/generate_learning_path_markdown.js#L37) with the language code of the articles for the appropriate section.
+Open a pull request for the change.
+1. Run **generate_learning_path_markdown.js** as described above.
+1. `cp -r learningpath/* <path-to-innersourcecommons.org-repo>/resources/learningpath/`.
```

Comment:
Missing a step - you need to update `resources/learningpath/index.md` in the innersourceommons.org repo with links to the new translation.

Reply:
Updated!  Have a look.